src/Base/index.js

```diff
@@ -36,42 +37,58 @@ const Input = ({
     }
 
     if (focused) {
-      return props.focusColor || '#52525B';
+      return props.focusColor || props.borderColor || '#52525B';
```

Comment:
@Dhruv-Rathi I am not understand why we need two props and how do write documentation for this if another developer has to use. Why borderColor is needed, focusColor is enough right?

Reply:
@rjvim if there is no focus colour but it is focused then the borderColor should be props.borderColor.

Reply:
@Dhruv-Rathi Let's remove on of them. But i guess the terminology "focus" tells that that color comes when user focuses. I think we should call it - `focusBorderColor` -> merge them both plz